src/shop-app.html

```diff
@@ -250,6 +250,17 @@
         num-items="{{numItems}}"
         total="{{total}}"></shop-cart-data>
 
+    <!--
+      shop-payment-request provides the logic and hooks into the PaymentRequest API.
+    -->
+    <shop-payment-request
+        id="payment"
+        on-process-payment="_onProcessPayment"
+        on-error="_onPaymentError"
+        supported-methods='["visa", "mastercard"]'
+        android-pay='{"merchantName":"Android Pay Demo","merchantId":"12345678901234567890","environment":"TEST","allowedCardNetworks":["AMEX","DISCOVER","MASTERCARD","VISA"],"paymentMethodTokenizationParameters":{"tokenizationType":"GATEWAY_TOKEN","parameters":{"gateway":"stripe","stripe:publishableKey":"pk_live_abcdefghijklmnopqrstuvwx","stripe:version":"2016-07-06"}}}'>
```

Comment:
this is a comment for myself since that might require a bit of refactoring: this could be move to a config file.